checks.d/openstack.py

```diff
@@ -427,20 +499,21 @@ def get_server_stats(self):
         for sid in server_ids:
             self.get_stats_for_single_server(sid)
 
-    def get_all_server_ids(self):
-        url = '{0}/servers'.format(self._nova_url)
+    def get_all_server_ids(self, filter_by_host=None):
+        url = '{0}/servers{1}'.format(self._nova_url, "?host=%s" % filter_by_host if filter_by_host else '')
```

Comment:
If you are getting a list of all instances, you will need to have `all_tenants=1` in the url